packages/noise/package.json

```diff
@@ -26,7 +26,7 @@
   "devDependencies": {
     "@nomicfoundation/hardhat-chai-matchers": "^1.0.3",
     "@nomiclabs/hardhat-ethers": "^2.1.1",
-    "@types/chai": "^4.3.3",
+    "@types/chai": "^4.3.5",
```

Comment:
fun fact: vitest had a bug until ~a month ago where it would display the wrong color for expected/received data in failing tests, and just discovered it when writing the tests for this PR

bumping vitest to at least 0.31.1 fixes it, but includes/requires a `@types/chai` bump, which conflicts with the one here so I bumped this too

I'll prob pull this out into its own PR once I get closer to this PR being ready and take a clean up pass